### DISCUSSION TITLE
Truffle testing gone wrong

### DISCUSSION THREAD

**Discussion by mustafaassaad** [ConsenSys-archive/truffle#5513] [General Q&A]
Hey guys sorry I'm a beginner at smart contracts... 

I'm testing my contracts and I'm getting a few errors which I don't know the meaning of!

Note: I used the "Write a relatable NFT smart contract" recourses from truffle suite. https://trufflesuite.com/guides/rentable-nft/

See pic below 👇🏻
![image](https://user-images.githubusercontent.com/89585671/189127841-62a56405-ecfc-4008-bb23-63410558a4ab.jpeg)

**Comment by apdan510** [comment]
@mustafaassaad Any chance you can share a public repo? I want to attempt to reproduce this behavior on my end. 

Thank you!
